security/rustls/src/main.rs

```diff
@@ -47,5 +49,7 @@ async fn main() -> std::io::Result<()> {
     })
     .bind_rustls("127.0.0.1:8443", config)?
     .run()
-    .await
+    .await?;
+    println!("Started http server: 127.0.0.1:8443");
```

Comment:
this print needs to come before the server await

Reply:
Ah. I'm curious. Is that because it will not print? I had thought it was misleading printing before the server spawned. I'll certainly move it to after the validation!

Reply:
.await will resolve when the server is shutting down